edx_proctoring/callbacks.py

```diff
@@ -159,3 +160,24 @@ def get(self, request, attempt_code):  # pylint: disable=unused-argument
             },
             status=200
         )
+
+
+class StudentProctoredExamAttempt(APIView):
+    """
+    Test API, It will change the attempt status to 'submitted'
+    """
+
+    def get(self, request):  # pylint: disable=unused-argument
+        """
+        Returns the status of an exam attempt. Given that this is an unauthenticated
+        caller, we will only return the status string, no additional information
+        about the exam
+        """
+        return Response(
+            data={
+                # IMPORTANT: Don't add more information to this as it is an
+                # unauthenticated endpoint
+                'status': 'submitted',
+            },
+            status=200
+        )
```

Comment:
The idea here is that the page opened in the proctoring application will call back into the backend to tell the backend that it has shut down. We'll modify the status of the attempt to 'submitted' if it's in the 'waiting for app shutdown' state. This one is here for manual testing, but we'll probably merge it with the AttemptStatus endpoint, passing a query parameter
 ?action=shutdown.